tests/libcxx/expected_results.txt

```diff
@@ -640,6 +643,11 @@ std/ranges/range.access/rbegin.pass.cpp FAIL
 std/ranges/range.access/rend.pass.cpp FAIL
 std/ranges/range.access/size.pass.cpp FAIL
 
+# C5101: use of preprocessor directive in function-like macro argument list is undefined behavior

```

Comment:
I would like to see "warning" before C5101 here, for increased searchability. (Also `skipped_tests.txt`.)